robert_beverage/scripts/web_server

```diff
@@ -1,5 +1,29 @@
 #!/usr/bin/env python
 import rospy
+import socket
+import SocketServer
+import SimpleHTTPServer
+from std_msgs.msg import String
 
 rospy.init_node('web_server_node')
-rospy.loginfo('Inside web_server script')
\ No newline at end of file
+rospy.loginfo('Inside web_server script')
+
+HOST = socket.gethostname()
+PORT = 8080
+address = ("", PORT)
+
+class RequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+    def do_GET(self):
+        if self.path.startswith("/beverage"):
+            info = self.path.strip("/")[9:]
+            coor = dict(x.split("=")for x in info.split("&"))
+            rospy.loginfo("x=%s\ty=%s", coor['x'], coor['y'])
+        self.path = "/workspace/robert/base/src/robert/robert_beverage/scripts/index.html"
```

Comment:
please do not use hardcoded path please use this code http://wiki.ros.org/Packages#Client_Library_Support

Reply:
where do index.html have to be located for rospkg to find it?

Reply:
rospkg is not locating index.html itself. It is providing path to package `robert_beverage` and you could concatenate relative path to index.html with path discovered by rospkg.

Reply:
```
            self.path = rospack.get_path('robert_beverage') + "/scripts/index.html"
            rospy.loginfo(self.path)
        return SimpleHTTPServer.SimpleHTTPRequestHandler.do_GET(self)
```
In browser type: http://localhost:8080/
```
[INFO] [1524043827.425584]: /home/user/workspace/robert/base/src/robert/robert_beverage/scripts/index.html
127.0.0.1 - - [18/Apr/2018 09:30:27] code 404, message File not found
127.0.0.1 - - [18/Apr/2018 09:30:27] "GET / HTTP/1.1" 404 -
```
rospkg starts looking from /home, but request handler starts from /workspace, so that code isn;t working

Reply:
Please try to change current web server directory to package one found by rospack using this approach https://stackoverflow.com/a/39801780 and serve index.html using relative path inside the package.